docs/manual/security.md

```diff
@@ -84,27 +85,25 @@ redis.conf file. It should be long enough to prevent brute force attacks
 for two reasons:
 
 * Redis is very fast at serving queries. Many passwords per second can be tested by an external client.
-* The Redis password is stored inside the **redis.conf** file and inside the client configuration, so it does not need to be remembered by the system administrator, and thus it can be very long.
+* The Redis password is stored inside the **redis.conf** file and inside the client configuration. Since the system administrator does not need to remember it, the password can be very long.
```

Comment:
```suggestion
* The Redis password is stored in the **redis.conf** file and inside the client configuration. Since the system administrator does not need to remember it, the password can be very long.
```